spec.bs

```diff
@@ -2320,6 +2350,45 @@ a {{ReportingBrowserSignals}} |browserSignals|, and a [=direct from seller signa
      :: |reportingMacroMap|
 </div>
 
+# Reporting # {#reporting}
+
+## forDebuggingOnly ## {#for-debugging-only-header}
+
+*This first introductory paragraph is non-normative.*
+
+"`generateBid()`" and "`scoreAd()`" can call
+{{InterestGroupBiddingAndScoringScriptRunnerGlobalScope/forDebuggingOnly}}.{{ForDebuggingOnly/reportAdAuctionWin(url)}} and

```

Comment:
the "." there is not a period ending the sentence, it's `forDebuggingOnly.reportAdAuctionWin`

Reply:
I think this should be `{{InterestGroupBiddingAndScoringScriptRunnerGlobalScope/forDebuggingOnly}}'s {{ForDebuggingOnly/reportAdAuctionWin()}} and {{ForDebuggingOnly/reportAdAuctionLoss()}} methods`. It is a little more wordy, but that's how most specs have this structured, and it is semantically correct (since the period isn't in a weird place).

Maybe that can be done in a follow-up.

Reply:
Ok, changing this in the next replated PR (https://github.com/WICG/turtledove/pull/1023).